spec/issue_exporting_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe 'IssueExporting' do
+  describe '#turn_options_into_querystring' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.